Services/DiscordBot.cs

```diff
@@ -287,7 +294,7 @@ private async Task RecordScore(SocketMessage message)
                     var scoreChannel = _client.GetChannel(dmManifest.ScoreKeeperChannelId) as IMessageChannel;
                     if (scoreChannel != null)
                     {
-                        await scoreChannel.SendMessageAsync(message.Content.Replace(".report ", "").Replace("!report ", ""));
+                        await scoreChannel.SendMessageAsync(message.Content.Replace(".report ", "").Replace("!report ", "").Replace(".result ", "").Replace("!result ", ""));

```

Comment:
same with this with the rebase

Reply:
These aren't crucial things, so I'll probably just merge them and then add them back in later if I need to.